src/Microsoft.Health.Dicom.Core/Features/Model/VersionedInstanceIdentifier.cs

```diff
@@ -36,5 +36,5 @@ public override int GetHashCode()
         => base.GetHashCode() ^ Version.GetHashCode();
 
     public override string ToString()
-        => base.ToString() + $"Version: {Version}";
+        => base.ToString() + $", Version: {Version}";
```

Comment:
Are we outputting this somewhere? Surely a customer would have no idea what "version" here means

Reply:
No, I was doing that before. Just correcting it